eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -43,6 +43,13 @@ steps:
             /bl:${{ parameters.repoLogPath }}/restore2.binlog
     displayName: Restore solution
 
+  - pwsh: |
```

Comment:
consider: Add a comment that this step is here temporary until we get this to work reliably as part of the build?